packages/fast-usdc-contract/test/snapshots/fast-usdc.contract.test.ts.md

```diff
@@ -1622,9 +1633,13 @@ Generated by [AVA](https://avajs.dev).
         },
         LogStore_kindHandle: 'Alleged: kind',
         StateUnwrapper_kindHandle: 'Alleged: kind',
-        asyncFuncEagerWakers: [],
+        asyncFuncEagerWakers: [
+          Object @Alleged: asyncFlow flow {},
+        ],
         asyncFuncFailures: {},
-        flowForOutcomeVow: {},
+        flowForOutcomeVow: {
+          'Alleged: VowInternalsKit vowV0': 'Alleged: asyncFlow flow',
+        },
```

Comment:
I don't exactly understand what `asyncFuncEagerWakers` and `flowForOutcomeVow` but I suppose these are compatible changes.

and we do have a relevant upgrade test: `deploy HEAD; upgrade to support EVM destinations`

Reply:
They are internal state for async flow.